scripts/calc-impl-progress.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Script to calculate the implementation progress of API calls in the SnowballR Backend and update the wiki.
+ *
+ * Usage: node calc-impl-progress.js
```

Comment:
```suggestion
 * Usage: node scripts/calc-impl-progress.js
```

I would prefer this one, as the most users probably would try this command still being in the root directory, what do you think?

Reply:
Yes makes sense.